tests/v2/validation/provisioning/k3s/README.md

```diff
@@ -215,4 +216,59 @@ For custom clusters, the below config is needed, only AWS/EC2 will work.
     "awsUser": "ubuntu",
     "volumeSize": 50
   },
+```
+
+## Advanced Settings
+This encapsulates any other setting that is applied in the cluster.spec. Currently we have support for:
+* cluster agent customization 
+* fleet agent customization
+
+Please read up on general k8s to get an idea of correct formatting for:
+* resource requests
+* resource limits
+* node affinity
+* tolerations
+
+```json
+"advancedOptions": {
+    "clusterAgentCustomization": { // change this to fleetAgentCustomization for fleet agent
```

Comment:
Clarification: since you just change this to `fleetAgentCustomization`, I'm assuming the parameters will remain the same, correct?

Reply:
yes, the params are all the same!!